### DISCUSSION TITLE
Does A(utomated)E(mergency)B(roadcast)S(ystem) report on incoming helicopter events?

### DISCUSSION THREAD

**Discussion by sharpgolden** [TEHE-Studios/ExpandedHelicopterEvents#275] [Q&A] [answered]
So far(2 month into the game) I cant hear message about approaching helicopter on AEBS. In vanilla there is a message "Air activity is detected" warning you, but so far I cant hear it, is it absent or plays far closer to an actual event?

**Comment by Chuckleberry-Finn** [comment]
It plays a message yes.

**Reply by sharpgolden** [reply]
I see, is it a regular message or a custom one? Because either I didnt see it, or didnt realise that it was it.

**Comment by Chuckleberry-Finn** [comment]
Have you gotten any events?

**Reply by sharpgolden** [reply]
Hello, yes I had.(sorry for a late response) I had already played 3 games with at least 1 month in and its unclear that I had any premature warnings like in vanilla. There were some misc dialogs in AEBS, but I doubt they were it.

**Comment by sharpgolden** [comment, marked as answer]
Alright, I found the answer eventually. There is a custom message that warns you about an air activity before it actually happens.
![20240322014413_1](https://github.com/TEHE-Studios/ExpandedHelicopterEvents/assets/119094611/61fcf579-5cde-497e-b34b-a9a643608223)
Thank you Chuckberry for help :)
